fetch_cluster_logs.bash

```diff
@@ -101,20 +116,25 @@ else
   done
 fi
 
+if [[ -z $max_artifact_size ]]; then
+  # default to 2GB
+  max_artifact_size=2000
+fi
+
 ssh_options="-A -T -l $ssh_user -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
 if [[ ! -z $identity_file ]]; then
   ssh_options="-i $identity_file ${ssh_options}"
 fi
 
 # download the DC/OS CLI
 if [ ! -f dcos-cli ]; then
-  wget https://downloads.dcos.io/binaries/cli/linux/x86-64/dcos-1.12/dcos --output-document=dcos-cli
+  wget https://downloads.dcos.io/cli/releases/binaries/dcos/linux/x86-64/0.7.5/dcos --output-document=dcos-cli
```

Comment:
We *might* want to tie this to a variable / option if we want or need to use different versions?